contact.html

```diff
@@ -34,13 +34,20 @@ <h3>Damian's House of Wonders</h3>
 				</div>
 			</nav>	
 			<main>
-				
+				<img src="/Users/ioacademy/Documents/reductito.github.io/images/20171104_165743.jpg" />
+				<div>
+					<a href="mailto:damianbell96@gmail.com">Email me</a>
+					<a href="https://www.linkedin.com/in/damian-bell-79b14a148/">View My LinkedIn Profile</a>
```

Comment:
you could make this open in a new tab pretty easily (can't remember how) but it may help the ux